server/api/accounts/accounts.js

```diff
@@ -0,0 +1,29 @@
+import Q from 'q'
+import jsforce from 'jsforce'
+
+export default function fetchAccounts(connection) {
+  const deferred = Q.defer();
+  const records = [];
+
+  connection.bulk.query("Select Id, FirstName, LastName FROM Account")
```

Comment:
Maybe for the update/insert part. Since this query doesn't use any user supplied input i think it's fine :)